docker-container/launch-m1.sh

```diff
@@ -0,0 +1,28 @@
+source config.sh
+set -x
+# Clean environment from before
+rm -rf docker-magento
+
+python set-magento-version.py $m1Version
+docker-compose down -v
+docker-compose up -d
+sleep 5
+
+# Install Sample Data and Initialize Magento
+if [ "$installSampleData" = true ] ; then
+    docker exec -it magento1_store install-sampledata
+fi
+sleep 1
```

Comment:
why you need sleep?

Reply:
I sleep here because I found errors within the DB if the scripts were run too close to one another. Will make a comment within the file